### DISCUSSION TITLE
查看mod管理卡顿

### DISCUSSION THREAD

**Discussion by 4484fdfdfgre88** [Meloong-Git/PCL#4850] [提问] [answered]
查看mod管理卡顿

**Comment by tangge233** [comment]
Mod 列表太长会卡顿，图片加载出来也会卡顿（WPF 特性，修不了），这是正常情况，一般加载完后放一会就不卡了。
_如果可以的话，给 PCL 加一个懒加载？_

**Reply by MoYuan-CN** [reply]
dotNet Framework 的 Image 有原生 lazy-loading 么？没有的话手动处理很麻烦

**Comment by LTCatt** [comment, marked as answer]
#4854 ？
